CDS/scripts/js_utils/youtube/youtube-upload-lib.js

```diff
@@ -1,11 +1,13 @@
-var Promise = require('promise');
-var fs = require('fs');
-var youtubeAuth = require('./youtube-auth.js');
-var googleapis = require('googleapis');
-var OAuth2 = googleapis.auth.OAuth2;
-var dataStore = require('../Datastore');
+const Promise = require('promise');
+const fs = require('fs');
+const youtubeAuth = require('./youtube-auth.js');
+const googleapis = require('googleapis');
+const OAuth2 = googleapis.auth.OAuth2;
+const dataStore = require('../Datastore');
+const https = require('https');
 
 const YOUTUBE_API_VERSION = 'v3';
+const POSTER_IMAGE_DOWNLOAD_DIR = '/tmp';
```

Comment:
would prefer that this is a default that can be changed in-route - e.g., when we deploy to AWS would want this on instance storage if it's available